manifests/config.pp

```diff
@@ -161,7 +161,7 @@
     content => template($config),
     owner   => '0',
     group   => $rabbitmq_group,
-    mode    => '0640',
+    mode    => '0644',
```

Comment:
If the directories themselves are readable by the group, as are the files, I think these files can (and should?) stay 0640?

Reply:
I have my doubts, too. On the one hand, SSL certificates should not be readable by everyone, but on the other hand these defaults are given by the SGID bit via /etc/rabbitmq. 

Maybe this question can be answered by someone who has more experience with RabbitMQ?

Reply:
What do you mean given by the sgid bit on the directory? You can have files with mode 0640 in a directory that’s world readable or executable w sgid bit (say 2755 or 2751). Sgid bit just affects default group ownership of files created in the directory afaik